### ISSUE TITLE
evaluateCurrentRoute does not work with defaultChild

### ISSUE THREAD

**Issue by ArtskydJ** [TehShrike/abstract-state-router#59]
1. `app` state has the `defaultChild`: `default-child-state`
2. `app` state has a `template`
3. call `stateRouter.evaluateCurrentState('app')` on an invalid route
4. white screen of death
5. call `stateRouter.evaluateCurrentState('app.default-child-state')` on an invalid path, and  `app.default-child-state` is loaded

**Comment by TehShrike** [comment]
hmm, I couldn't reproduce.  I tried:

```js
const createStateRouter = require('abstract-state-router')
const Ractive = require('ractive')
const ractiveRenderer = require('ractive-state-router')(Ractive)
const stateRouter = createStateRouter(ractiveRenderer, 'body')

stateRouter.addState({
  name: 'parent',
  route: '/parent',
  defaultChild: 'default-child',
  template: '<h1>top</h1><ui-view></ui-view>'
})

stateRouter.addState({
  name: 'parent.default-child',
  route: '/default-child',
  template: '<h2>default child</h2><ui-view></ui-view>'
})

stateRouter.addState({
  name: 'parent.other-child',
  route: '/other-child',
  template: '<h2>other child</h2><ui-view></ui-view>'
})

stateRouter.evaluateCurrentRoute('parent')
```

**Comment by TehShrike** [comment]
abstract-state-router's package.json does currently call for hash-brown-router `~1.5.0`. A [bug with applying default routes was fixed in 1.5.1](https://github.com/TehShrike/hash-brown-router/commit/8eca5dfb4a22469fd8a96b2d42d0b4cfc21b45c1), so make sure you've got that patch.

**Comment by ArtskydJ** [comment]
I was using abstract-state-router@5. I updated to the latest asr/hbr, and that seems to have fixed the issue.
